app/models/component/manifest.rb

```diff
@@ -49,6 +51,16 @@ def labels(prefix = Configurations.kubernetes.label_prefix)
         "#{prefix}/nodetype" => node.id }.with_indifferent_access
     end
 
+    def annotations
+      app_string = "[{\"source\":\"#{datadog_logs_source}\",\"service\":\"#{datadog_logs_service}\"}]"
```

Comment:
Instead of having to write these backslashes (`\`) everywhere, what do you think of writing something like:

```ruby
require 'json'

# ...
app_string = [{ source: datadog_logs_source,
                service: datadog_logs_service }]
# ...
app_string.to_json
```

Reply:
This is not a Ruby hash... this is a string that must to conform to that format otherwise datadog agent will not recognise the fields.

Reply:
But for all intents and purposes, doesn't my suggestion produce that very same _string_ but with a syntax that's easier on the eyes?